JonArnfred/week1/assignment1/HelloWorld_sans_react.html

```diff
@@ -0,0 +1,108 @@
+
+<html>
+<head>
+	<title>Hello world without react</title>
+	<style type="text/css">
+	body {
+		width: 1000px;
+		margin: auto;
+	}
+	</style>
+</head>
+<body>
+
+<!-- I assumed you wanted a form, since action is a form property, not a button property
+https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button. However, I rewrote the template markup a bit-->
+  <form id="myform">
+	  <input type="text" value="warnings">
+	  <input type="submit" value="Submit"/>
+  </form>
+	<input type="text" onKeyDown="checkInputNoWarning()" value="no warnings">
+  <button onclick="printem()">All messages so far</button> 
```

Comment:
I know there is no reason for a form. However, you wrote: "2. Find a way to submit and print the message -->". If I read 'submit' I think form. Also in the original markup the button had an action property which is a form property. However I explained part of this in a comment "I assumed you wanted a form, since action is a form property, not a button property".